scripts/ccl

```diff
@@ -8,43 +10,45 @@
 # Any definition of CCL_DEFAULT_DIRECTORY already present in the
 # environment takes precedence over definition made below.
 
-if [ -z "$CCL_DEFAULT_DIRECTORY" ]; then
+if [ -z "${CCL_DEFAULT_DIRECTORY:-}" ]; then
   CCL_DEFAULT_DIRECTORY=/usr/local/src/ccl
 fi
 
-# If you don't want to guess the name of the lisp kernel on
+# If you don't want to guess the name of the OpenMCL kernel on
 # every invocation (or if you want to use a kernel with a
 # non-default name), you might want to uncomment and change
 # the following line:
 #OPENMCL_KERNEL=some_name
 
-if [ -z "$OPENMCL_KERNEL" ]; then
-  case `uname -s` in
-    Darwin) case `arch` in
-              ppc*) OPENMCL_KERNEL=dppccl ;;
-              i386) OPENMCL_KERNEL=dx86cl ;;
-            esac ;;
-    Linux) case `uname -m` in
-              ppc*) OPENMCL_KERNEL=ppccl ;;
-              *86*) OPENMCL_KERNEL=lx86cl ;;
-              *arm*) OPENMCL_KERNEL=armcl ;;
-	      *aarch64*) OPENMCL_KERNEL=armcl ;;
-           esac ;;
+if [ -z "${OPENMCL_KERNEL:-}" ]; then
+  case "$(uname -s)" in
+    Darwin) case "$(arch)" in
+      ppc*) OPENMCL_KERNEL=dppccl ;;
+      i386) OPENMCL_KERNEL=dx86cl ;;
+      *)
+        echo "Unsupported architecture"
+        exit 1 ;;
+      esac ;;
+    Linux)
```

Comment:
There might be an indentation issue here (or GitHub is not displaying things correctly).